modules/ROOT/pages/clustering/disaster-recovery.adoc

```diff
@@ -69,30 +69,33 @@ See xref:clustering/setup/routing.adoc#clustering-routing[Server-side routing] f
 === Start the Neo4j process
 
 ==== Objective
+
 ====
-The Neo4j process is started on all servers which are not _lost_.
+The Neo4j process is started on all servers that are not _lost_.
 ====
 
 ==== Path to correct state
+
 Start the Neo4j process on all servers that are _offline_.
 If a server is unable to start, inspect the logs and contact support personnel.
 The server may have to be considered indefinitely lost.
 
 [[make-the-system-database-write-available]]
-=== Make the `system` database write available
+=== Make the `system` database write-available
 
 ==== Objective
 ====
-The `system` database is write available.
+The `system` database is able to accept write operations.
 ====
 
 The `system` database contains the view of the cluster.
 This includes which servers and databases are present, where they live and how they are configured.
 During a disaster, the view of the cluster might need to change to reflect a new reality, such as removing lost servers.
 Databases might also need to be recreated to regain write availability.
-Because both of these steps are executed by modifying the `system` database, making the `system` database write available is a vital first step during disaster recovery.
+Because both of these steps are executed by modifying the `system` database, making the `system` database write-enabled is a vital first step during disaster recovery.
```

Comment:
This is in your hands, but I don't think it nice to change from a technically correct wording which we have defined in the beginning of the document to multiple adjacent ones which are not defined. Like write-enabled, able to accept write operations and write-capable.